packages/usage-examples/src/3-create-declarative-request.ts

```diff
@@ -0,0 +1,110 @@
+/**
+ * Creating a declarative Request using a public gateway.
+ * ======================================================
+ *
+ * In this example we will create a declarative request, meaning there is no manual payment detection, the payment information update is manual.
+ * The request will be created at a node running locally (https://docs.request.network/request-protocol/getting-started-1/deploy-a-request-node).
+ */
+
+/**
+ * IMPORTS
+ *
+ * First we import the 2 packages we will need to create the request:
+ */
+// The signature provider allow us to sign the request
```

Comment:
Should we add a small sentence to explain that they can use another (even develop) signature provider.